README.md

```diff
@@ -1,2 +1,233 @@
-# messaging
-Messaging for Web3
+# Anonymous Communication Protocol
+
+Messaging for Web3.
+
+## Motivation
+In the current decentralised application landscape, we are seeing projects
+struggle to achieve mainstream adoption. We believe that the underlying
+protocols do not have sufficient capability to enable the necessary level of
+adoption. Part of the problem is that these applications often require the
```

Comment:
This section might be too blockchain focused, perhaps tying it into Anonymous Communication literature would include Secure Messaging projects like Briar etc (see https://hackmd.io/2MDws3OvSZ2wz-ULI6tpLQ#Secure-messaging---Non-ethereum for list of such projects).

Reply:
I'll try to write something not so audience specific..